### ISSUE TITLE
Version 3.0.1 missing in github

### ISSUE THREAD

**Issue by lo1tuma** [jquense/date-math#5]
On current master the version of this package defined in [`package.json`](https://github.com/jquense/date-math/blob/6764271be83bc8d60bcf89cec48064dc4526f1f2/package.json#L3) is `3.0.0`. But on npm the latest version is `3.0.1`.

```
$ npm view date-arithmetic versions dist-tags
versions = [ '1.0.0', '2.0.0', '3.0.0', '3.0.1' ]
dist-tags = { latest: '3.0.1' }
```

Are there any changes in this version compared to `3.0.0`? If so can those changes be published to github?

**Comment by lo1tuma** [comment]
Seems like #1 has never been published. I can’t find the changes in the published version `3.0.0` nor `3.0.1` on npm.

**Comment by jquense** [comment]
Oops, Im traveling but I'll try and get it first thing tomorrow, feel free to ping me if I forget
